src/app/static/src/app/store/root/actions.ts

```diff
@@ -52,5 +57,28 @@ export const actions: ActionTree<RootState, RootState> & RootActions = {
 
     async changeLanguage(context, payload) {
         await changeLanguage<RootState>(context, payload)
+    },
+
+    async fetchADRKey(context) {
+        await api<RootMutation, RootMutation>(context)
+            .ignoreErrors()
```

Comment:
Does this still need to be changed to withError?

Reply:
I think if fetching the API key fails there's not much point in complaining about it as the user can't really take another action to rectify it? Unlike saving/deleting where they can try again.

Reply:
Fair enough!